src/NServiceBus.Core/Persistence/Learning/SagaPersister/LearningSagaPersistence.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace NServiceBus.Features
+{
+    using System;
+    using System.IO;
+    using NServiceBus.Sagas;
+
+    class LearningSagaPersistence : Feature
+    {
+        internal LearningSagaPersistence()
+        {
+            DependsOn<Sagas>();
+            Defaults(s => s.Set<ISagaIdGenerator>(new LearningSagaIdGenerator()));
+            Defaults(s => s.SetDefault(StorageLocationKey, Path.Combine(AppDomain.CurrentDomain.BaseDirectory, ".sagas")));
```

Comment:
:-1: that would make the path to long issues we already have worse and its very likely that sagas is the only thing this class of persistence would ever have sine the LearningTransport is sepf sufficient and outbox imo doesn't make sense to provide since users would have no way to store their business data along side the sagas and outbox record